src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -688,6 +697,97 @@ class ContentSidebar extends PureComponent<Props, State> {
         );
     };
 
+    /**
+     * Re-formats file collaborators response
+     *
+     * @private
+     * @param {Array<User>} contacts - Collection of collaborated users
+     * @return {SelectorItems}
+     */
+    parseContacts = (contacts: Array<User>): SelectorItems =>
+        contacts.map((collab: User) => {
+            const { id, name, login } = collab;
+            return {
+                id,
+                name,
+                item: { ...collab, email: login }
+            };
+        });
+
+    /**
+     * File approver contacts fetch success callback
+     *
+     * @private
+     * @param {BoxItemCollection} data - Collaborators response data
+     * @return {void}
+     */
+    getApproverContactsSuccessCallback = (data: UserCollection): void => {
+        const contacts = data.entries || [];
+        this.setState({ approverSelectorContacts: this.parseContacts(contacts) });
+    };
+
+    /**
+     * File @mention contacts fetch success callback
+     *
+     * @private
+     * @param {BoxItemCollection} data - Collaborators response data
+     * @return {void}
+     */
+    getMentionContactsSuccessCallback = (data: UserCollection): void => {
+        const contacts = data.entries || [];
+        this.setState({ mentionSelectorContacts: this.parseContacts(contacts) });
+    };
+
+    /**
+     * File @mention contacts fetch success callback
+     *
+     * @private
+     * @param {API} api - Box API instance
+     * @param {string} searchStr - Search string to filter file collaborators by
+     * @param {Function} successCallback - Fetch success callback
+     * @return {void}
+     */
+    getApproverWithQuery = debounce((searchStr: string): void => {
```

Comment:
When do these 2 get called? On every keydown in the textarea but then is debounced here?